src/util/cfUtil.ts

```diff
@@ -43,9 +43,9 @@ export default class CFUtil {
     static async createService(params: ICreateServiceInstanceParams) {
         log.verbose(`Creating a service instance with parameters: ${JSON.stringify(params)}`);
         const resources = await this.requestCfApi(`/v3/service_plans?names=${params.planName}&space_guids=${params.spaceGuid}`);
-        const publicPlan = resources.find(resource => resource.visibility_type === "public");
+        const publicPlan = resources.find(resource => ["public", "organization"].includes(resource.visibility_type));
```

Comment:
Didn't we agree that you will no longer check on this but use the service name?

Reply:
@sap/cf-tools need plan guid for instance creation: https://github.com/SAP/cloud-foundry-tools-api/blob/edc848eaeb45ac4405b02e8f4f7a25e5047e43f0/src/cf-local.ts#L448

Reply:
Yes of course but please find the plan via the owning service and check its name and not by checking the visibilty.

Reply:
Also the current implementation could use a wrong plan, if a completely different service would have a plan with the same name